addon/components/tf-toggle-button.js

```diff
@@ -0,0 +1,69 @@
+/**
+  @module ember-ticketfly-toggle-button
+ */
+import Component from 'ember-component';
+import layout from '../templates/components/tf-toggle-button';
+import get from 'ember-metal/get';
+import computed from 'ember-computed';
+
+/**
+  colorGroup options: 'neutral'
+  sizeGroup options: 'primary', 'secondary'
+
+  @public
+  @class TfToggleButton
+  @extends Ember.Component
+ */
+export default Component.extend({
+  layout,
+
+  tagName: 'button',
+  type: 'button',
+  attributeBindings: ['disabled', 'aria-label', 'aria-pressed', 'type'],
+  classNameBindings: ['buttonCategory', 'disabled:c-tf-toggle-button--disabled', 'pressed:c-tf-toggle-button--pressed'],
+  classNames: ['c-tf-toggle-button', 'u-pv-0', 'u-ph-xs'],
+
+  hook: 'tf-toggle-button',
+
+  onClick() { return this; }, //noop
+
+  click() {
+    this.toggleProperty('pressed');
+    const pressed = get(this, 'pressed');
+    get(this, 'onClick')(pressed);
+  },
+
+  pressed: false,
+
+  // aria-pressed is not a boolean value, but a string token of 'true' or 'false'
+  "aria-pressed": computed('pressed', {
```

Comment:
I might be totally wrong, but _I think_ we can camelize this attribute to be `ariaPressed` and it "just works". If I am wrong, single quotes here?